app/views/layouts/mailer.html.haml

```diff
@@ -0,0 +1,6 @@
+!!!
+%html
+  %head{:lang => "en"}
```

Comment:
Correct me if I'm wrong, but I thought in HTML5, the `lang` attribute goes on the HTML tag, right?